src/features/collapsible-sidebar/CollapsibleSidebarLogo.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import styled from 'styled-components';
+
+import Logo from '../../icon/logo/BoxLogo';
+import PlainButton from '../../components/plain-button/PlainButton';
+import LinkBase from '../../components/link/LinkBase';
+import IconHamburger from '../../icons/general/IconHamburger';
+
+import CollapsibleSidebarItem from './CollapsibleSidebarItem';
+import './CollapsibleSidebarLogo.scss';
+
+const StyledLogo = styled(Logo)`
+    & path,
+    & .fill-color {
+        fill: ${props => props.theme.primary.foreground};
+    }
+`;
+
+const StyledIconHamburger = styled(IconHamburger)`
+    & .fill-color {
+        fill: ${props => props.theme.primary.foreground};
+    }
+`;
+
+type Props = {
+    /** optional badge to be displayed next to logo */
+    badge?: React.Node,
```

Comment:
This is the main difference between EUA and BUIE implementation.
Turning it to be a generic node so the component doesn't need EUA implementation details and admin console can also leverage it as well.